layout/FilteredBooleanQuery.js

```diff
@@ -53,9 +53,6 @@ Layout.prototype.render = function( vs ){
   // handle sorting views under 'sort'
   if( this._sort.length ){
     this._sort.forEach( function( view ){
-      if( !Array.isArray( q.sort ) ){
```

Comment:
was this no longer necessary? i'm not sure what it was doing to begin with, but interesting to understand why it isn't needed.